src/dev/env.md

```diff
@@ -0,0 +1,21 @@
+# Environment Variables
+
+There are a number of environment variables which control the behaviour of the
+yk system.
+
+## `YK_PRINT_IR`
+
+When `YK_PRINT_IR=1`, yk will print (to `stderr`) a textual version of each
+trace's LLVM IR prior to code generation.
+
+This variable is set by the `c_tests` suite for structural IR testing.
```

Comment:
I don't think this sentence is useful: it suggests *only* the test suite should use this, but people might find other uses for it.

Reply:
That implication wasn't intentional. Shall we just remove it then, or?

Reply:
Agreed.

Reply:
Ping on this one.